test/test_cli.py

```diff
@@ -1,7 +1,8 @@
 import subprocess
 
+
 def test_cli_hello() -> None:
-    # Run the CLI using 'uv' to ensure correct environment
-    result = subprocess.run(["foamadapter"], capture_output=True, text=True)
+    result = subprocess.run(
+        ["foamadapter", "solver", "--help"], capture_output=True, text=True
```

Comment:
I would argue better rename now 
```suggestion
        ["neofoam", "solver", "--help"], capture_output=True, text=True
```

Reply:
i will start the renaming after the first iteration